b2sdk/stream/progress.py

```diff
@@ -63,6 +58,11 @@ def read(self, size=None):
         self._progress_update(len(data))
         return data
 
+    def seek(self, pos, whence=0):
+        pos = super(ReadingStreamWithProgress, self).seek(pos, whence=whence)
+        self.bytes_completed = pos
```

Comment:
Given that this was the source of a bug, it would be nice to have a comment here saying why 
it's appropriate to set `bytes_completed` here, but not in `AbstractStreamWithProgress`.

Reply:
added comment